client/src/components/helpers/link.tsx

```diff
@@ -1,14 +1,11 @@
-import { Link as GatsbyLink } from 'gatsby';
+import { GatsbyLinkProps, Link as GatsbyLink } from 'gatsby';
 import React from 'react';
 
-interface LinkProps {
+interface LinkProps extends Omit<GatsbyLinkProps<string>, 'ref'> {
```

Comment:
```suggestion
interface LinkProps extends Omit<GatsbyLinkProps<Record<string, unknown>>, 'ref'> {
```
The 'state' type needs to be broad enough to allow for objects like

```ts
{ breadcrumbBlockClick: string }
```

Reply:
oh sorry about that 😅, this was me messing around seeing if there is another place except `breadcumb`. forgot to update it 😓